### ISSUE TITLE
Unable to get the changelog/release notes working on the release page

### ISSUE THREAD

**Issue by yrshaikh** [cycjimmy/semantic-release-action#51]
**Describe the bug**
Hi, I am setting up semantic release on my project, I am able to get it working but I have an issue with the changelog/release-notes. It does not show up like how it's supposed to show.

Current behavior -

![image](https://user-images.githubusercontent.com/2232423/104843636-422b3f00-58fe-11eb-9dec-a8e1fd8e72ea.png)

**Workflow**
I have configured the required extra plugins as mentioned in the docs, here is the link to my [workflow file](https://github.com/yrshaikh/good-urls/blob/main/.github/workflows/build_pack_publish_main.yml) on GitHub - 

```
name: Build pack and publish main
on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:

  check_version_number:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Get semantic release version
        uses: cycjimmy/semantic-release-action@v2
        id: semantic_dry
        with:
          branch: main
          dry_run: true
          extra_plugins: |
            @semantic-release/changelog
            @semantic-release/git
            @semantic-release/github
        env:
          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
    outputs:
      new_release_published: ${{ steps.semantic_dry.outputs.new_release_published }}
      new_release_version: ${{ steps.semantic_dry.outputs.new_release_version }}

  build_package_push:
    needs: check_version_number
    if: needs.check_version_number.outputs.new_release_published == 'true'
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          submodules: 'recursive'
      
      - name:  Setup dotnet
        uses: actions/setup-dotnet@v1
        with:
            dotnet-version: '5.0.x'
      
      - name: Build project
        run: dotnet build --configuration Release -p:Version=${{ needs.check_version_number.outputs.new_release_version }}.0
      
      - name: Package
        run: dotnet pack --configuration Release --output ./artifacts -p:PackageVersion=${{ needs.check_version_number.outputs.new_release_version }}      

      - name: Upload artifact
        uses: actions/upload-artifact@v2
        with:
          name: nuget-artifact
          path: ./artifacts/*.nupkg
          if-no-files-found: error

  tag_version:
    needs: [build_package_push,check_version_number]
    if: needs.check_version_number.outputs.new_release_published == 'true'
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Get semantic release version
        uses: cycjimmy/semantic-release-action@v2
        id: semantic
        with:
          branch: main
          extra_plugins: |
            @semantic-release/changelog
            @semantic-release/git
            @semantic-release/github
        env:
          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
      - name: Echo logs
        run: |
          echo ${{ steps.semantic.outputs.new_release_version }}
          echo ${{ steps.semantic.outputs.new_release_major_version }}
          echo ${{ steps.semantic.outputs.new_release_minor_version }}
          echo ${{ steps.semantic.outputs.new_release_patch_version }}
  push_to_nuget:
    needs: [tag_version,check_version_number]
    if: needs.check_version_number.outputs.new_release_published == 'true'
    runs-on: ubuntu-latest
    steps:
    - name: Download artifact
      uses: actions/download-artifact@v2
      with:
        name: nuget-artifact
    - run: ls -R
    - uses: actions/setup-dotnet@v1
      with:
        dotnet-version: '5.0.x' # SDK Version to use.
    - name: Upload to nuget
      run: dotnet nuget push ./*.nupkg --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
```

**Expected behavior**
I want the changelog to be as how most repositories (including this one) shows as highlighted in this screenshot below 

![image](https://user-images.githubusercontent.com/2232423/104843659-57a06900-58fe-11eb-874a-423d22afe5e0.png)

**Additional context**
N/A

**Comment by cycjimmy** [comment]
`@semantic-release/github` is already part of `semantic-release` and does not have to be installed separately.

**Comment by yrshaikh** [comment]
@cycjimmy can you please explain what change do I have to do here? I did not follow your comment, thanks in advance.

**Comment by yrshaikh** [comment]
I will try to remove it and try, hope that it the issue and fixes it.

**Comment by yrshaikh** [comment]
@cycjimmy I tried [removing the extra plugin](https://github.com/yrshaikh/good-urls/commit/49fec22b973dfe5c724fb9f98c15ae1b82a57942), but still didn't get the expected release notes as shown in the screenshot in the issue thread.

Below is the logs 

```

28s
Run cycjimmy/semantic-release-action@v2
  with:
    branch: main
    dry_run: true
    extra_plugins: @semantic-release/changelog
  @semantic-release/git
  
  env:
    GITHUB_TOKEN: ***
added 2 packages in 0.214s

[4:39:10 PM] [semantic-release] › ℹ  Running semantic-release version 17.3.6
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/changelog"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/git"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/changelog"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/git"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[4:39:11 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[4:39:13 PM] [semantic-release] › ⚠  Run automated release from branch main on repository yrshaikh/good-urls.git in dry-run mode
[4:39:13 PM] [semantic-release] › ✔  Allowed to push to the Git repository
[4:39:13 PM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/changelog"
[4:39:13 PM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/changelog"
[4:39:13 PM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/git"
[4:39:13 PM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/git"
[4:39:13 PM] [semantic-release] › ℹ  Start step "verifyConditions" of plugin "@semantic-release/github"
[4:39:13 PM] [semantic-release] [@semantic-release/github] › ℹ  Verify GitHub authentication (https://api.github.com)
[4:39:13 PM] [semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/github"
[4:39:13 PM] [semantic-release] › ℹ  Found git tag v1.1.10 associated with version 1.1.10 on branch main
[4:39:13 PM] [semantic-release] › ℹ  Found 3 commits since last release
[4:39:13 PM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: fix: add another fake commit message
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: fix: add fake commit message
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The release type for the commit is patch
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: chore: remove /github plugin
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
[4:39:13 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analysis of 3 commits complete: patch release
[4:39:13 PM] [semantic-release] › ✔  Completed step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
[4:39:13 PM] [semantic-release] › ℹ  The next release version is 1.1.11
[4:39:13 PM] [semantic-release] › ⚠  Skip step "prepare" of plugin "@semantic-release/changelog" in dry-run mode
[4:39:13 PM] [semantic-release] › ⚠  Skip step "prepare" of plugin "@semantic-release/git" in dry-run mode
[4:39:13 PM] [semantic-release] › ⚠  Skip v1.1.11 tag creation in dry-run mode
[4:39:13 PM] [semantic-release] › ⚠  Skip step "publish" of plugin "@semantic-release/github" in dry-run mode
[4:39:13 PM] [semantic-release] › ⚠  Skip step "success" of plugin "@semantic-release/github" in dry-run mode
[4:39:13 PM] [semantic-release] › ✔  Published release 1.1.11 on default channel
[4:39:13 PM] [semantic-release] › ℹ  Release note for version 1.1.11:
```

**Comment by cycjimmy** [comment]
There is no problem with the log. Check the permissions of your GitHub token.
